packages/legend-query/src/stores/QueryBuilderPropertyEditorState.ts

```diff
@@ -68,13 +68,17 @@ export const getPropertyChainName = (
         SUPPORTED_FUNCTIONS.SUBTYPE,
       )
     ) {
-      propertyNameChain.unshift(
-        `(@${
-          currentExpression.parametersValues.filter(
-            (param) => param instanceof InstanceValue,
-          )[0]?.genericType?.value.rawType.name
-        })`,
-      );
+      const propertyName =
```

Comment:
I'd like to refrain from making comments on codestyle, but could we just inline both of these constants? Or maybe keep one, they are not reused anywhere below, we can inline them to `propertyNameChain.unshift(...)`